Things.md

```diff
@@ -178,4 +180,16 @@ Classes are generated using [ObjectMakr](https://github.com/FullScreenShenanigan
 
 When a FullScreenShenanigans game starts up, ObjectMakr receives the inheritance structure and properties list from an `objects.js` file. It then generates prototypes for all of the internal classes (Box, Tree, etc.). Most of the basic properties, like `height`, `width`, and `opacity` are defined in the parent Thing class. ObjectMakr's `addThing` is a shortcut to initializing a new instance of a Thing class and assigning properties to it in one step. 
 
-More can be read about ObjectMakr on its [ReadMe](https://github.com/FullScreenShenanigans/ObjectMakr/blob/master/README.md).
\ No newline at end of file
+More can be read about ObjectMakr on its [ReadMe](https://github.com/FullScreenShenanigans/ObjectMakr/blob/master/README.md).
+
+# Sprites
+
+Sprites comprise any visual you see in a FullScreenShenanigans game. They are stored as string representations of an image, containing information about every pixel. The game generates the image from this string in real-time.
+
```

Comment:
Good first summary. Can you link to the PixelRendr and PixelDrawr readmes here? You should be able to rely on them to document the specifics.